best-practices-cel/restrict-node-port/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: restrict-node-port
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ../restrict-node-port.yaml
+    - patch:
+        resource:
+          apiVersion: kyverno.io/v1
+          kind: ClusterPolicy
+          metadata:
+            name: restrict-nodeport
+          spec:
+            validationFailureAction: Enforce
+    - assert:
+        file: chainsaw-step-01-assert-1.yaml
+  - name: step-02
+    try:
+    - apply:
+        file: ../../../best-practices/restrict-node-port/.chainsaw-test/good-services.yaml
```

Comment:
I used to reference resources for pod security cel policies as well.
For example: https://github.com/kyverno/policies/blob/main/pod-security-cel/baseline/disallow-capabilities/.chainsaw-test/chainsaw-test.yaml#L26
Should I create the resources instead?

Reply:
I used to reference resources for pod security cel policies as well.
For example: https://github.com/kyverno/policies/blob/main/pod-security-cel/baseline/disallow-capabilities/.chainsaw-test/chainsaw-test.yaml#L26
Should I create the resources instead?

Reply:
The test manifests in [pod-security-cel](https://github.com/kyverno/policies/tree/main/pod-security-cel) follow the same approach. I think it has to be changed there too?

Reply:
I see. Yes, I think it would be better if we kept these policies hermetically sealed allowing easy transport if needed. When we create these types of links, should we move them later for a different system of organization all the tests will break.